cl/users/email_handlers.py

```diff
@@ -16,29 +16,30 @@
 from django.utils.timezone import now
 
 from cl.users.models import (
-    OBJECT_TYPES,
-    SUB_TYPES,
-    BackoffEvent,
+    EMAIL_NOTIFICATIONS,
+    FLAG_TYPES,
+    STATUS_TYPES,
     EmailFlag,
     EmailSent,
+    FailedEmail,
 )
 
 
-def get_bounce_subtype(event_sub_type: str) -> int:
+def get_bounce_subtype(notification_subtype: str) -> int:
```

Comment:
Rather than do this, I remembered that we have cl.lib.model_helpers.make_choices_group_lookup. You can use that to invert a choices group so that you can look up the values more easily. Take a look and see if it eliminates this function.

Reply:
I couldn't use the make_choices_group_lookup because seems it only inverted tuples like:
`(("Judge", ((JUDGE, "Judge"))))` for tuples like `(key, 'value')` it only transforms to a dict.

So based on that function I created a similar one: `invert_choices_group_lookup` that takes (key:, 'value')  and changes to {'value': key}

So I deleted the previous function: `get_bounce_subtype` and use something like: 
`EMAIL_NOTIFICATIONS.INVERTED[notification_subtype]`